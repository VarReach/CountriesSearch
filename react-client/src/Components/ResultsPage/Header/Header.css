.results__header {
  position: relative;
  background-color: #141414;
  height: 8em;
  border-bottom: 2px solid #141414;
}

.results__header .results__search {
  height: 100%;
  display: flex;
  align-items: center;
}

.results__header .logo {
  position: relative;
  height: 4em;
  width: 4em;
  bottom: initial;
  margin: 0 0.6em 0 1.4em;
}