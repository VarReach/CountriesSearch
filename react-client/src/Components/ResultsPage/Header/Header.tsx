import React, { FC } from "react";
import "./Header.css"
// Components
import SearchBar from "../../SearchBar/SearchBar";
/* @ts-ignore */
import globe from "../../../Assets/globe.png";

export const Header: FC = () => {
  return (
    <div className="results__header">
      <div className="results__search">
        <img className="logo" src={globe} />
        <SearchBar 
          page={"results"}
        />
      </div>
    </div>
  );
}

export default Header;