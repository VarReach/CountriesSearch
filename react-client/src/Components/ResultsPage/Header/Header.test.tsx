import React from "react";
import { render } from "@testing-library/react";
// Components
import Header from "./Header";

/**
 * Covers tests for the ResultsPage Header
 */

describe("<Header/>", () => {
  test("renders without crashing", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});