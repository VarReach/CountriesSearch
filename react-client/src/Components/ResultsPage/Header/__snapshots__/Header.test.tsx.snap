// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders without crashing 1`] = `
<div>
  <div
    class="results__header"
  >
    <div
      class="results__search"
    >
      <img
        class="logo"
        src="test-file-stub"
      />
      <form
        class="search_form search_form__results"
        id="search_form__results"
      >
        <input
          autocapitalize="off"
          autocomplete="off"
          autocorrect="off"
          class="search_form__input"
          id="search_form__input_results"
          tabindex="1"
          value=""
        />
        <button
          class="search_form__clear search_form__button hide"
          id="search_form__clear"
          type="button"
        >
          <i
            class="material-icons"
          >
            clear
          </i>
        </button>
        <div
          class="search_form__inputs"
        >
          <button
            class="search_form__type-button search_form__button"
            id="search_form__type-button_results"
            type="button"
          >
            <span>
              name
            </span>
            <i
              class="material-icons"
            >
              expand_more
            </i>
          </button>
          <button
            class="search_form__submit search_form__button"
            id="search_form__submit_results"
          >
            <i
              class="material-icons"
            >
              search
            </i>
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
